signup/forms.py

```diff
@@ -0,0 +1,26 @@
+from django import forms
+from django.forms import ModelForm
+from django.contrib.auth.forms import UserCreationForm
+from .models import User
+from django.core.exceptions import ValidationError
+import datetime
```

Comment:
`ValidationError`は使用していないので削除しましょう！
そしてimportの記述順として，

1. 標準ライブラリ
2. サードパーティに関連するもの
3. ローカルな アプリケーション/ライブラリ に特有のもの

と並べましょう。具体的には，
```suggestion
import datetime

from django import forms
from django.contrib.auth.forms import UserCreationForm
from django.forms import ModelForm

from .models import User
```
こんな感じです。
VSCodeを設定すると，保存するだけで並べ替えられるよう自動化できるのでお勧めです！
https://zenn.dev/wtkn25/articles/python-isort

ついでに余力があればですが，[ここなど](https://itstudio.co/2020/09/26/11077/#toc3)を参照してオートフォーマットを設定するとさらに便利ですのでお勧めです。
Pythonのコーディング規則にもし興味があったら，PEP8で調べてみてください。